static/app/components/charts/utils.tsx

```diff
@@ -279,3 +285,98 @@ export const processTableResults = (tableResults?: TableDataWithTitle[]) => {
 export const getPreviousSeriesName = (seriesName: string) => {
   return `previous ${seriesName}`;
 };
+
+function formatList(items: Array<string | number | undefined>) {
+  const filteredItems = items.filter(item => !!item);
+  return [[...filteredItems].slice(0, -1).join(', '), [...filteredItems].slice(-1)]
+    .filter(type => !!type)
+    .join(' and ');
+}
+
+export function useAria(
```

Comment:
Nit: could have a more descriptive name like `useEchartsAriaLabels` or something

Reply:
Not even a nit, definitely worth changing!

Reply:
Ahh great point! Renamed in https://github.com/getsentry/sentry/pull/39653/commits/4076bf61bdf4fe218e4e9c50858bdae4c7d58bb0